sonar-plugin-api/src/main/java/org/sonar/api/web/WidgetProperties.java

```diff
@@ -22,6 +22,10 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
+ * @deprecated since 6.2, as dashboards have been removed
+ */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 public @interface WidgetProperties {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)